### PR TITLE
IOS-296 - Show confirmation alert when accepting payment request.

### PR DESCRIPTION
**Pull request by roberthein**
Add `approvePaymentForIndexPath(_:)` and `declinePaymentForIndexPath(_:)`.
Re-route cell delegate trough alert to ☝️these new methods.
Add formatted localization.

img_3867

### CODE REVIEW

File: Toshi/Controllers/Messaging/ChatController.swift [hunk 1]
```diff
@@ -571,6 +571,33 @@ final class ChatController: OverlayController {
 
         showMediaPickerBlock(nil)
     }
+    
+    func approvePaymentForIndexPath(_ indexPath: IndexPath) {
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatController.swift]: those should be fileprivate ?

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatController.swift]: Added fileprivate.

Comment (yuliaveres) [reply Toshi/Controllers/Messaging/ChatController.swift]: also let me know what do you think about this, please:
https://swift.org/documentation/api-design-guidelines/#naming

`func approvePayment(for indexPath: IndexPath)` then `approvePayment(for: indexPath)` vs `approvePaymentForIndexPath(indexPath)` ?

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatController.swift]: `approvePayment(for indexPath: IndexPath)` causes a error in this case.

Comment (yuliaveres) [reply Toshi/Controllers/Messaging/ChatController.swift]: ok, got it

**Pull request merged**